packages/react-menu/src/Menu.stories.tsx

```diff
@@ -93,7 +102,7 @@ export const CustomTrigger = () => {
       <button onClick={() => setOpen(true)}>Custom Trigger</button>
```

Comment:
Why is this button here? The point of this story is the `CustomMenuTrigger`, right?